rtwo/models/provider.py

```diff
@@ -283,6 +283,17 @@ def get_driver(self, identity, **provider_credentials):
         self.identity = identity
         self.lc_driver = OpenStack_Esh_NodeDriver
         self.set_options(provider_credentials)
+        if 'ex_force_auth_token' in self.options.keys() and 'ex_force_base_url' in self.options.keys():
```

Comment:
This isn't necessary -- `ex_force_auth_token` and `ex_force_base_url` are found in the lines *just below this* L297:306